clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/security/config/CredentialsConfig.java

```diff
@@ -150,6 +149,7 @@ public void setDefaultResult(String defaultResult) {
         private String assumeRole;
         private String sessionName;
         private List<LifecycleHook> lifecycleHooks;
+        private Boolean allowLaunchDescription;
```

Comment:
You need to name this the same as the attribute on NetflixAmazonCredentials (it gets ObjectMapper.convertValue'd over there)